explorer/site-config/hca-dcp/dev/index/projectsEntityConfig.ts

```diff
@@ -90,6 +91,15 @@ export const projectsEntityConfig: EntityConfig = {
         id: HCA_DCP_CATEGORY_KEY.PROJECT_TITLE,
         width: { max: "2fr", min: "374px" },
       },
+      {
+        componentConfig: {
+          component: BioNetworkCell,
```

Comment:
Let's add `BioNetworkCell` to the component `index.ts` so we can use the same style here as elsewhere `C.BioNetworkCell`.